modules/tensor_mechanics/doc/content/source/materials/HillElastoPlasticityStressUpdate.md

```diff
@@ -0,0 +1,129 @@
+\end{equation}
+
+With $f$ being the yield condition, $\boldsymbol{s}$ the deviatoric stress and $\gamma$ the plastic multiplier, the plastic strain rate is given as:
+
+\begin{equation}
+\dot{\boldsymbol{\epsilon}}^{\boldsymbol{p}} = \dot{\gamma}\frac{\partial f}{\partial \boldsymbol{s}}
+\label{eq:normality_condition}
+\end{equation}
+
+[eq:normality_condition] is a statement of normality condition, i.e., the associative flow rule. Using [eq:yield_condition], the associative flow rule in [eq:normality_condition] can be written as:
+
+\begin{equation}
+\dot{\boldsymbol{\epsilon}}^{\boldsymbol{p}} = \dot{\gamma} \boldsymbol{A} \boldsymbol{s}
+\label{eq:normality_condition2}
+\end{equation}
+
+which can be written in incremental form as:
+
+\begin{equation}
+\Delta\boldsymbol{\epsilon}^{\boldsymbol{p}} = \Delta\gamma \boldsymbol{A} \boldsymbol{s}
+\label{eq:normality_condition2_incremental}
+\end{equation}
+
+If $\alpha$ is the internal state variable associated with isotropic hardening, and $\beta$ is the energy conjugate to the $\dot{\alpha}$, the $\dot{\alpha}$ is written as:
+
+\begin{equation}
+\dot{\alpha} = - \dot{\gamma}\frac{\partial f}{\partial \beta}
+\label{eq:alpha_dot}
+\end{equation}
+
+The thermodynamic variable $\beta$ is $s_y$ here (see [!cite](versino2018generalized) for details). Equation [eq:alpha_dot] can be written in integrated incremental form as:
+
+\begin{equation}
+\Delta\alpha = 2 s_y \Delta\gamma
+\label{eq:delta_alpha}
+\end{equation}
+
+Using [eq:delta_alpha] in [eq:normality_condition2_incremental], taking the partial derivative of $\Delta\boldsymbol{\epsilon}^{\boldsymbol{p}}$ with respect to $\boldsymbol{s}$, and applying chain rule, we obtain:
+
+\begin{equation}
+\frac{\partial \Delta\boldsymbol{\epsilon}^{\boldsymbol{p}}}{\partial \boldsymbol{s}} = \frac{\partial \Delta\boldsymbol{\epsilon}^{\boldsymbol{p}}}{\partial \Delta\alpha} \frac{\partial \Delta\alpha}{\partial \boldsymbol{s}} = \frac{\boldsymbol{A} \boldsymbol{s}}{2s_y} \frac{\partial \Delta\alpha}{\partial \boldsymbol{s}}
+\label{eq:inverse_slope}
+\end{equation}
+
+\begin{equation}
+\Rightarrow \frac{\partial \boldsymbol{s}}{\partial \Delta\boldsymbol{\epsilon}^{\boldsymbol{p}}} = \frac{2s_y}{\boldsymbol{A} \boldsymbol{s}} \frac{\partial \boldsymbol{s}}{\partial \Delta\alpha}
+\label{eq:slopeA}
+\end{equation}
+
+For the case where flow rule involves linear hardening with hardening constant $K$:
+
+\begin{equation}
+\frac{\partial \boldsymbol{s}}{\partial \Delta\alpha} = K
+\label{eq:slopeB}
+\end{equation}
+
+Substituting [eq:slopeB] in [eq:slopeA] we obtain:
+
+\begin{equation}
+\Rightarrow \frac{\partial \boldsymbol{s}}{\partial \Delta\boldsymbol{\epsilon}^{\boldsymbol{p}}} = \frac{2s_y}{\boldsymbol{A} \boldsymbol{s}} K
+\label{eq:slopeC}
+\end{equation}
+
+The left hand side of [eq:slopeC] gives the slope of the stress vs plastic strain curve. This equation can be written for the direct compoents of stress ($s_ii$) as:
```

Comment:
It's great that this got written out!  `s_ii` -> `s_{ii}`

Reply:
Fixed